tascrum/serializers.py

```diff
@@ -614,8 +654,8 @@ class Meta:
         fields = ['id', 'labels']
 
     def get_labels(self, obj):
-        label = obj.boardl.all()
-        return LabelTimelineSerializer(label, many=True).data
```

Comment:
this function will returns all boards in database. and it's wrong.

Reply:
it will be handled